README.md

```diff
@@ -113,7 +142,7 @@ Double Deuce<br>![Double Deuce](double2.png) | <pre>var d2Die = new WeightedDie(
 - [ ] More real world examples
 - [x] Percentages/ratios for weight
 - [ ] Include common dice like the examples (another package)
-- [ ] Built-in labels
+- [x] Built-in labels
```

Comment:
I took the liberty to mark this feature as complete, although I'm not really sure if I really added built-in labels the way you would like them.